DirectProgramming/DPC++/StructuredGrids/particle-diffusion/src/motionsim.cpp

```diff
@@ -37,6 +36,8 @@
 
 using namespace sycl;
 using namespace std;
+namespace oneapi {}
+using namespace oneapi;
```

Comment:
Remove all these `using namespace` in a tutorial.
https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice

Reply:
Thank you for your input on this, we are discussing this internally and will get back to you.